spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "create" do
+    let(:user) {User.create(:first_name => "Rebecca", :last_name => "Holzschuh", :email => "rebeccaholzschuh@gmail.com", :username=> "rebecca")}
+    it "adds one user to the database" do
```

Comment:
Your first 2 tests are frivolous.  You are literally testing FactoryGirl.create; which is not useful to your test of the "create" method of the controller.